app/models/event.rb

```diff
@@ -0,0 +1,20 @@
+class Event < ActiveRecord::Base
+  belongs_to :user
+
+  enum event_type: {
+    account_created: 1,
+    phone_confirmed: 2,
+    password_changed: 3,
+    phone_changed: 4,
+    email_changed: 5,
+    authenticator_enabled: 6,
+    authenticator_disabled: 7
+  }
+
+  validates :event_type, presence: true
+
+  def event_type_to_s
+    raise 'Invalid event type' unless Event::event_types.keys.include?(event_type)
```

Comment:
You are right, this was from a previous version where the event_type was passed as an argument.